<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGCSkyeTestMotorsPanel</class>
 <widget class="QWidget" name="QGCSkyeTestMotorsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="labelThrust">
     <property name="text">
      <string>Thrust i</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="SliderThrust">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="doubleSpinBoxThrust"/>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelOrientation">
     <property name="toolTip">
      <string extracomment="512*84 = 43008 is a quarter turn (90Â°)"/>
     </property>
     <property name="text">
      <string>Orientation i in degree</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDial" name="dialOrientation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="minimum">
      <number>-86016</number>
     </property>
     <property name="maximum">
      <number>86016</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBoxOrientation">
     <property name="minimum">
      <number>-86016</number>
     </property>
     <property name="maximum">
      <number>86016</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
